templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/terraform/vm_config.sh

```diff
@@ -13,10 +13,12 @@ sudo adduser xrdp ssl-cert
 # Required packages for Docker installation
 sudo apt-get install ca-certificates curl gnupg lsb-release
 # Get Docker Public key from Nexus
-curl -fsSL ${nexus_proxy_url}/repository/docker-public-key/gpg | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/docker-archive-keyring.gpg
+# shellcheck disable=SC2154
+curl -fsSL "${nexus_proxy_url}"/repository/docker-public-key/gpg | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/docker-archive-keyring.gpg
```

Comment:
Is _nexus_proxy_url_ optional? If not I prefer you check you got a value instead of ignoring the warning.
```bash
${nexus_proxy_url:?}
```

Reply:
Done